sources/it/regions.hjson

```diff
@@ -0,0 +1,180 @@
+{
+  versions: '>=1'
+  type: http
+  production: true
+  countryCode: IT
+  wikidata: Q16110
+  note: Italy regions
+  data: https://sophox.org/regions/geojson.json
+  attribution: [
+    {
+      label: {
+        en: © OpenStreetMap contributors
+      }
+      url: {
+        en: https://www.openstreetmap.org/copyright
+      }
+    }
+    {
+      label: {
+        en: Elastic Maps Service
+      }
+      url: {
+        en: https://www.elastic.co/elastic-maps-service
+      }
+    }
+  ]
+  fieldMapping: [
+    {
+      type: id
+      name: iso_3166_2
+      desc: ISO-3166-2 identifier
+    }
+    {
+      type: id
+      name: istat
+      desc: ISTAT ID
+    }
+    {
+      type: property
+      name: label_en
+      desc: Province name (English)
+    }
+    {
+      type: property
+      name: label_it
+      desc: Province name (Italian)
```

Comment:
The `v2` manifest still uses the `desc` field for the field name. So let's change these accordingly. Versions >=`v6.6` use the localized field names.

```suggestion
      desc: Region name (Italian)
```